content/pages/community.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Community</title>
+        <meta name="save_as" content="community/index.html">
+        <meta name="url" content="community/">
+        <meta name="body_class" content="community_page">
+    </head>
+
+    <body>
+        <div class="community secondary">
+            <div class="col-12">
+                <h1>Community</h1>
+                <p>
+                    Kiwi TCMS is developed in the open. Here are some of the channels we use
+                    to communicate and contribute:
+                </p>
+                <p>
+                    <strong>IRC:</strong>
+                    <a href="https://kiwiirc.com/client/irc.freenode.net/kiwitcms"><code>#kiwitcms</code></a> on <a href="http://freenode.net/">irc.freenode.net</a>
+                    (for the easiest start, <a href="https://kiwiirc.com/client/irc.freenode.net/kiwitcms">join via Kiwi IRC</a>)
+                </p>
+                <p>
+                    <strong>Twitter:</strong>
+                    <a href="https://twitter.com/KiwiTCMS">@KiwiTCMS</a>
+                </p>
+                <p>
+                    <strong>Issue tracker:</strong> Use the GitHub issue tracker
+                    for the various <a href="https://github.com/kiwitcms">Kiwi TCMS
+                    repositories</a> to file bugs and features request.
+                    If you need support, please send your questions to the
+                    <a href=" https://github.com/kiwitcms/Kiwi/issues">kiwitmcs-users</a>
+                    issue tracker rather than filing a GitHub issue.
```

Comment:
Issue tracker is at https://github.com/kiwitcms/Kiwi/issues, no mailing list. Please update the entire text to reflect that.